lib/open_food_network/enterprise_fee_applicator.rb

```diff
@@ -20,6 +20,7 @@ def create_adjustment(label, adjustable)
       AdjustmentMetadata.create! adjustment:, enterprise: enterprise_fee.enterprise,
                                  fee_name: enterprise_fee.name, fee_type: enterprise_fee.fee_type,
                                  enterprise_role: role
+      adjustment
```

Comment:
Good idea, it would be pretty confusing to call create_adjustment and receive an AdjustmentMetadata object instead!